arrows/core/detected_object_set_input_csv.cxx

```diff
@@ -234,6 +297,51 @@ get_input()
 
 
 // ------------------------------------------------------------------
+void
+detected_object_set_input_csv::priv::
+parse_detection( const std::vector< std::string >& parsed_line)
+{
+  kwiver::vital::detected_object_type_sptr dot;
+
+  // Create DOT object if classifiers are present
+  if ( parsed_line.size() > 7 )
+  {
+    dot = std::make_shared<kwiver::vital::detected_object_type>();
+    const size_t limit( parsed_line.size() );
+
+    for (size_t i = 7; i < limit; i += 2 )
```

Comment:
Very minor nit, but why does i start at 7? I'm sure there is a good reason, but it's not clear from looking at the code. Could you add a comment or perhaps initialize a variable with a descriptive name which tells the reader what it is and use that to initialize i. Either way is ok.